scripts/pt_br_new.py

```
@@ -0,0 +1,125 @@
+    ]
+    }}}}
+    ```
+
+IMPORTANT: Only output the translated texts and nothing else, no need explaination or instruction. The input text is between "=== BEGIN OF TEXT ===" and "=== END OF TEXT ===".
+
+Please translate the following texts to Brazilian Portuguese:
+
+=== BEGIN OF TEXT ===
+{{content}}
+=== END OF TEXT ===
+'''.strip()
```

Ruff (0.11.9)

Do not assign a `lambda` expression, use a `def`

Rewrite `prompt` as a `def`

(E731)